src/core/Akka.FSharp/Spawning.fs

```diff
@@ -0,0 +1,140 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="Spawning.fs" company="Akka.NET Project">
+//     Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2015 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+namespace Akka.FSharp
+
+open Akka.Actor
+open System
+open Microsoft.FSharp.Quotations
+open Microsoft.FSharp.Linq.QuotationEvaluation
+
+[<RequireQualifiedAccess>]
+module Configuration = 
+    /// Parses provided HOCON string into a valid Akka configuration object.
+    let parse = Akka.Configuration.ConfigurationFactory.ParseString
+    
+    /// Returns default Akka configuration.
+    let defaultConfig = Akka.Configuration.ConfigurationFactory.Default
+    
+    /// Loads Akka configuration from the project's .config file.
+    let load = Akka.Configuration.ConfigurationFactory.Load
+
+module System = 
+    /// Creates an actor system with remote deployment serialization enabled.
+    let create (name : string) (config : Akka.Configuration.Config) : ActorSystem = 
+        let system = ActorSystem.Create(name, config)
+        Serialization.exprSerializationSupport system
+        system
+
+[<AutoOpen>]
+module Spawn = 
```

Comment:
Concerning first (moving options forward on function arg list) - IMO this is bad. 

- First you have a set of similar functions, each using same subset of function arguments, but depending on the function they are placed in different order (i.e. if I have `spawn <system> <name> <actor-logic>` setting, other spawn functions where this order is broken looks counter-intuitive to me).
- Second, I think it's a good practice (especially from partial application perspective) that function args should be in order from most general to most specialized. In that case list of spawn options is in fact more specific than actor behavior.

About `Spawn` module. The only reason for it is to keep code in order. Spawn naming conventions are borrowed from Erlang and used as well in Scala's akka-typed package. In your example accidental using simple `open Spawn` would end with making function names unreasonable. This could be fixed by using [RequireQualifiedAccess](https://msdn.microsoft.com/en-us/library/ee340423.aspx) attribute. But again - it's breaking existing API in heavier way, than proposed changes, while possible gains are questionable.